libraries/radpdfprocessing/features/embedded-file-streams/embedded-file-streams.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Overview
+page_title: Overview
+slug: radpdfprocessing-embedded-file-streams-overview
+tags: pdf, file, stream, embedded
+position: 0
+---
+
+# Embedded File Streams - Overview
+
+As of **2024 Q1** RadPdfProcessing allows embedding files streams into the document. Thus, the content of the referenced files is embedded directly within the body of the PDF file.
+
+## The EmbeddedFile Class
+
+RadFixedDocument stores the integrated files in an **EmbeddedFilesCollection** accessed by the **EmbeddedFiles** property. Each **EmbeddedFile** contains **Name** (string) and **Data** (byte[]) properties. The specified Name represents the textual description of the embedded file, which can be displayed in the user interface of a viewer application.  The Data stores the byte[] of the file stream. 
```

Comment:
Maybe add "(it should be unique for the whole PDF document)" at the end of the sentence describing the Name property.